DevCycleTests/Models/DVCClientTest.swift

```diff
@@ -170,7 +172,67 @@ class DVCClientTest: XCTestCase {
         client.refetchConfig(sse: true, lastModified: 789)
         XCTAssertEqual(service.numberOfConfigCalls, 6)
     }
+    
+    func testSseCloseGetsCalledWhenBackgrounded() {
+        let service = MockService()
+        let user1 = try! DVCUser.builder().userId("user1").build()
+        let client = try! DVCClient.builder().user(user1).environmentKey("my_env_key").build(onInitialized: nil)
```

Comment:
for these tests and the one above, can you just use the `self.builder` that I set up? It makes sure that the requests service is correctly mocked out so that we aren't sending real API calls